pom.xml

```diff
@@ -503,5 +504,51 @@ limitations under the License.
         </plugins>
       </build>
     </profile>
+    <profile>
```

Comment:
Please separate this out into a separate PR

In general we're using Spotless for formatting, and I'm not sure we want to introduce another formatting step.

Reply:
Can you comment on how this is resolved? I don't think these changes are related to Guava.